app/commands/operations/__init__.py

```diff
@@ -0,0 +1,73 @@
+from app.commands import Command
```

Comment:
I think each of these operations again can be distinguished. But I want to know your opinion on when to decide to modularize and when they can be coupled.

Thanks